specification/relationships/Shop.json

```diff
@@ -31,6 +31,48 @@
           }
         }
       }
+    },
+    "default_product": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "object",
+          "required": [
+            "type",
+            "id"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "type": {
+              "$ref": "../types/Product.json"
```

Comment:
`Product` is geen resource / endpoint. Misschien toch maar `default_service` van maken?
We zouden in het nieuwe zending form in principe altijd alvast het `parcel` product kunnen selecteren. Mss later nog een default van maken als we het formulier helemaal af hebben.
Of toch default_carrier, dat we dan altijd de `parcel` service van die carrier selecteren?